linera-execution/src/lib.rs

```diff
@@ -414,6 +416,9 @@ pub trait ServiceRuntime: BaseRuntime {
         queried_id: UserApplicationId,
         argument: Vec<u8>,
     ) -> Result<Vec<u8>, ExecutionError>;
+
+    /// Get a blob of bytes from an arbitrary URL.
+    fn get_blob_from_url(&mut self, url: &str) -> Result<Vec<u8>, ExecutionError>;
```

Comment:
Optional: I'm wondering if `download_blob` is clearer :thinking:

Reply:
Right I thought about that too - but I wanted to disambiguate this from the upcoming Blob API, so `get_blob_from_url` seemed more clear in that respect.

Reply:
Do we need to use "blob" though? (could also be a file or content or bytes) We may use the word blob for a specific meaning in the core protocol soon